src/adapter/sources.ts

```diff
@@ -295,7 +295,7 @@ export class SourceContainer {
     if (ref.sourceReference)
       return this._sourceByReference.get(ref.sourceReference);
     if (ref.path)
-      return this._sourceByAbsolutePath.get(ref.path);
+      return this._sourceByAbsolutePath.get(utils.lowerCaseInsensitivePath(ref.path));
```

Comment:
Similar thing here. Without this we weren't marking breakpoints as resolved since source lookups failed.